src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  height: 70px;
+  margin: 20px 10px;
+}
+
+.search-form--small {
+  height: 42px;
+}
+
+.input {
+  background: transparent;
+  flex: 1;
+  border: 0;
+  outline: 0;
+  padding: 25px 17px;
+  color: #3d4e61;
+}
+
+.input--small {
+  padding: 11px 9px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.input--small::placeholder {
+  font-size: 14px;
+}
+
+.search-button {
+  border: 0;
+  cursor: pointer;
+  background-color: #fff;
+  margin-left: 26px;
+}
+
+.search-button--small {
+  margin-left: 13px;
+}
+
+.search-icon {
+  width: 19px;
+  height: 19px;
+}
+
+.search-icon--small {
+  width: 11px;
+  height: 11px;
+}
+
+.search-form:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
```

Comment:
you have to apply focus styles to the element that is full-width to make it look proper

Reply:
I did this task for 3 days. I completely understand your comment. I didn´t find a solution to this problem.